src/main/java/com/teragrep/aer_02/EventDataConsumer.java

```diff
@@ -146,7 +146,7 @@ public Long getValue() {
         SyslogMessage syslogMessage = new SyslogMessage()
                 .withSeverity(Severity.INFORMATIONAL)
                 .withFacility(Facility.LOCAL0)
-                .withTimestamp(enqueuedTime.toInstant())
+                .withTimestamp(enqueuedTime == null ? ZonedDateTime.now().toInstant() : enqueuedTime.toInstant())
```

Comment:
Does the `ZonedDateTime.now()` provide the system time zone, or should it be locked to a single time zone?

Reply:
It does indeed use the system timezone, and provided that EnqueuedTime is in UTC, the alternative time should also be in UTC. Changing to Instant.now().